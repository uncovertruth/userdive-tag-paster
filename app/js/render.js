/* @flow */
'use strict'
import Vue from 'vue'
import Info from '../components/info'

export default (userData: object, onClick: Function) => {
  const vm = new Vue({
    el: '#info',
    methods: {},
    render: h => {
      return h(
        Info,
        {
          props: {
            pageInfo: userData
          }
        }
      )
    }
  })
  vm.$on('changeStatus', () => onClick())
}
