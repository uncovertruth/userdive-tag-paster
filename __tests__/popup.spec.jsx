/* @flow */
/** @jsx h */
import { h } from 'preact'
import render from 'preact-render-to-string'

import Popup from '../app/components/popup'

describe('popup', () => {
  test('vaild id', () => {
    const tree = render(
      <Popup
        isActive
        data={{
          a: 'a',
          b: 1
        }}
      />
    )
    expect(tree).toMatchSnapshot()
  })
})
