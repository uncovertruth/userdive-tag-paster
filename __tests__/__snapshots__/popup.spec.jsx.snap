// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`popup vaild id 1`] = `"<div><table><tr><th>a</th><th>a</th></tr><tr><th>b</th><th>1</th></tr></table><button>Turn OFF</button></div>"`;
